source/includes/migrationName-description.rst

```diff
@@ -0,0 +1,4 @@
+Starting in ``mongosync`` 1.8.0, you can set a migration name for a sync
+operation. For example, you can set a migration name to identify the
+sync operations from multiple source clusters into one destination
```

Comment:
```suggestion
operation. For example, you can set a migration name to identify each
sync operation from multiple source clusters into one destination
```

Reply:
The current phrasing is a bit unclear on whether a single migration name applies to all sync operations into one destination or each sync operation representing different source clusters.

Reply:
Done.